docs/guides/operations-guide/package-upgrades.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: Package Upgrades
+---
+
+# Package Upgrades
+
+## Compute Nodes
+
+Set HOST environment to be used in subsequent commands to the compute node which is going to be updated
+
+```
+HOST=""
+```
+
+Disable compute host, so that no new instances are scheduled to it and instance state remains unchanged
+
+```
+osism manage compute disable $HOST
+```
+
+Live-migrate all instances to other compute nodes
+
+```
+osism manage compute migrate --yes $HOST
+```
+
+List remaining nodes in non-live-migratable state
+
+```
+osism manage compute list $HOST
+```
+
+Move shut off nodes to other nodes
+
+```
+INTERACTIVE=false openstack server list --all-projects --status SHUTOFF --host $HOST -f value -c ID | while read -r ID; do openstack server migrate $ID; done
```

Comment:
We should integrate this in the OSISM CLI as well. Please open an issue for this.

Reply:
Ack, I already started doing this while waiting for package upgrades/reboots.

https://github.com/osism/issues/issues/1248